src/core/render/index.js

```diff
@@ -251,29 +251,26 @@ export function Render(Base) {
 
     _renderSidebar(text) {
       const { maxLevel, subMaxLevel, loadSidebar, hideSidebar } = this.config;
+      const sidebarEl = dom.getNode('aside.sidebar');
+      const sidebarToggleEl = dom.getNode('button.sidebar-toggle');
 
       if (hideSidebar) {
-        // FIXME : better styling solution
-        [
-          document.querySelector('aside.sidebar'),
-          document.querySelector('button.sidebar-toggle'),
-        ]
-          .filter(e => !!e)
-          .forEach(node => node.parentNode.removeChild(node));
-        document.querySelector('section.content').style.right = 'unset';
-        document.querySelector('section.content').style.left = 'unset';
-        document.querySelector('section.content').style.position = 'relative';
-        document.querySelector('section.content').style.width = '100%';
```

Comment:
Why remove this?

Reply:
@trusktr --

From what I can tell these lines are unnecessary. Sidebar menu behavior is the same on desktop and mobile for the live docsify site and the preview for this PR. My guess is that setting style properties directly on the `section.content` element was a holdover from an older implementation that never got cleaned up.

Is there something I'm missing?